src/client/routes/events/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { EventUpdate, AddOrUpdateEventMutationFn } from './ManageEventTypes';
+
+export function updateEventsHandler(
+	events: null | EventUpdate[],
+	addOrUpdateEventFunction: AddOrUpdateEventMutationFn
+): string[] {
+	if (events) {
+		events.map(event =>
+			addOrUpdateEventFunction({
+				variables: {
+					input: {
+						...event,
+					},
+				},
+			})
+		);
+	}
+	return [];
```

Comment:
so this is going to return [] no matter what?

Reply:
My mistake, forgot to return something. The return type from the addOrUpdateEventFunction mutation is quite byzantine, i'm reverting this to return a list of event names.